### DISCUSSION TITLE
Binary payload

### DISCUSSION THREAD

**Discussion by Shesoff** [ddosify/ddosify#31] [Q&A] [answered]
Hi. 
How I can generate post request with `multipart/form-data` content-type? With classic curl I use next construction:
```
curl -X 'POST' 'http://myserver.mydomain.ru/api/v1/recognize/fast' \
  -H 'accept: application/json' \
  -H 'Content-Type: multipart/form-data' \
  -F 'img=@image.jpg;type=image/jpeg' \
  -F 'params={
  "bboxes": [
    {
      "id": 0,
      "bbox": {
        "left": 60,
        "right": 394,
        "top": 351,
        "bottom": 733
      }
    }
  ]
}'
```
that send json body and image.jpg file located nearby. Can I repeat this test by ddosify?

**Comment by kursataktas** [comment]
Hi @Shesoff 

Thanks for raising this. `multipart/form-data` will be supported in the future.

**Reply by Shesoff** [reply]
Thank you. 
Will be waiting.

**Comment by kursataktas** [comment, marked as answer]
Hi @Shesoff 

`multipart/form-data` support added to [Ddosify 0.7.0](https://github.com/ddosify/ddosify/releases/tag/v0.7.0). You can use it like explained [here](https://github.com/ddosify/ddosify#user-content-#payload_multipart).
